### PR TITLE
fix: rename the button 'Rerun' to 'Run' on the b2i page #1543

### LINKED ISSUE #1543: Suggest that rename the button 'Rerun' to 'Run' on the b2i page
**What would you like to be added**:

Suggest that rename the button 'Rerun' to 'Run' on the b2i page.

image

**Why is this needed**:

There are no differences between `Run` and `Rerun` according to the context here. And `Rerun` makes people feel confused.

/area devops
/good-first-issue

### PR DESCRIPTION
**Pull request by TinaSprunt**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
rename the button 'Rerun' to 'Run' on the b2i page #1543

**Which issue(s) this PR fixes**:
Fixes  #1543

**Special notes for reviewers:**
```
```
**Additional documentation, usage docs, etc.:**
```
```
/assign @xuliwenwenwen

### CODE REVIEW

File: src/pages/projects/containers/ImageBuilder/Detail/index.jsx [hunk 1]
```diff
@@ -91,8 +91,8 @@ export default class ImageBuilderDetail extends React.Component {
 
   getOperations = () => [
     {
-      key: 'reRun',
-      text: t('Rerun'),
+      key: 'Run',
+      text: t('run'),
```
Comment (harrisonliu5) [inline src/pages/projects/containers/ImageBuilder/Detail/index.jsx:95]: There is no 'run' in locales language, only have the 'Run'.

**Review by harrisonliu5 — CHANGES_REQUESTED** [review]

File: src/pages/projects/containers/ImageBuilder/Detail/index.jsx (hunk 1 above)
Comment (TinaSprunt) [reply src/pages/projects/containers/ImageBuilder/Detail/index.jsx:95]: thanks，I got it

**Comment by TinaSprunt** [comment]
/ assign @xuliwenwenwen
/assign @harrisonliu5

**Pull request merged**